src/ContactPage/Contact.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import "./Contact.scss";
+
+class Contact extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      name: "",
+      email: "",
+      subject: "",
+      message: "",
+      errors: {
+        name: "",
+        email: "",
+        subject: "",
+        message: ""
+      }
+    };
+  }
+
+  handleChange = event => {
+    event.preventDefault();
+    const validEmailRegex = RegExp(
+      /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i
+    );
+
+    const { name, value } = event.target;
+    let errors = this.state.errors;
+
+    switch (name) {
+      case "name":
```

Comment:
You've got 2 things here with the name of _name_. Possibly change one of these so its not confusing?